bpf/complexity-tests/510/bpf_host/6.txt

```diff
@@ -37,3 +37,4 @@
 -DENABLE_IPSEC=1
 -DEVENTS_MAP_RATE_LIMIT=1000
 -DENABLE_LOCAL_REDIRECT_POLICY=1
+-DENABLE_SERVICE_PROTOCOL_DIFFERENTIATION=1
```

Comment:
Shouldn't we add it to all complexity test cases and also to the MAX_*_OPTIONS in the Makefile?

Reply:
> Do you mean other the bpf_host test cases? I see the option in complexity test cases for other bpf_xdp programs.

I think it should be in basically every complexity test file. It makes a small difference in functions `lb{4,6}_lookup_service` and they are called from `nodeport_lb{4,6}`. These four functions are called from bpf_lxc, bpf_host, bpf_sock, bpf_xdp, bpf_overlay, and bpf_wireguard.

Reply:
Yes, sg! I decided to create separate commits in case